src/anndata/tests/helpers.py

```diff
@@ -494,7 +494,7 @@ def assert_equal_cupy(a, b, exact=False, elem_name=None):
 def assert_equal_ndarray(a, b, exact=False, elem_name=None):
     b = asarray(b)
     if not exact and is_numeric_dtype(a) and is_numeric_dtype(b):
-        assert a.shape == b.shape, format_msg(elem_name)
+        assert a.shape == b.shape, (a.shape, b.shape)
```

Comment:
I would like to keep the functionality where the format_message is included. Otherwise it can be difficult to figure out which element the error is coming from.